libgloss/arc/hl/configure.in

```diff
@@ -0,0 +1,57 @@
+AC_PREREQ(2.59)
```

Comment:
The same question, why do we need an autoconf system here. Why you don't use the one level up system?

Reply:
We can, but I think we should implement hl subdir in the same way as dw_uart.

Reply:
Not really, please check my comment on makefile.in. The idea is to keep the implementation simple, without extra work and efficient. Removing autoconf you remove 3.5K line of code which does nothing extra.

Reply:
done!